Application Actions/iTerm/1.0/default.json

```diff
@@ -0,0 +1,8 @@
+{
+  "RotateLeft"  : "Scroll Down",
+  "RotateRight" : "Scroll Up",
+  "SwipeUp"     : "Previous Pane",
+  "SwipeDown"   : "Next Pane",
+  "SwipeLeft"   : "Previous Tab",
+  "SwipeRight"  : "Next Tab"
```

Comment:
To be consistent with existing applications (e.g. Safari, Chrome, Firefox), can we rename those to

    Select Previous Pane
    Select Next Pane
    Select Previous Tab
    Select Next Tab

Thanks!